python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -122,21 +125,89 @@ def get_runs(graphene_info, filters, cursor=None, limit=None):
 ]
 
 
+class AssetComputeStatus(Enum):
+    OLD = "OLD"
+    GOOD = "GOOD"
+    NONE = "NONE"
+    UNKNOWN = "UNKNOWN"
+
+
+def get_all_asset_nodes(graphene_info):
```

Comment:
Ahh you're right @clairelin135 - I'll swap this out for that one! It looks like it returns GrapheneAssetNodes instead of just the external_asset_node objects, but I think that's ok? I'm not sure how much overhead there is in building them.